code/_globalvars/configuration.dm

```diff
@@ -23,14 +23,14 @@ GLOBAL_VAR_INIT(Debug2, FALSE)
 
 //This was a define, but I changed it to a variable so it can be changed in-game.(kept the all-caps definition because... code...) -Errorage
 //Protecting these because the proper way to edit them is with the config/secrets
-GLOBAL_VAR_INIT(MAX_EX_DEVESTATION_RANGE, 3)
+GLOBAL_VAR_INIT(MAX_EX_DEVESTATION_RANGE, 15)
 GLOBAL_PROTECT(MAX_EX_DEVESTATION_RANGE)
-GLOBAL_VAR_INIT(MAX_EX_HEAVY_RANGE, 7)
+GLOBAL_VAR_INIT(MAX_EX_HEAVY_RANGE, 35)
 GLOBAL_PROTECT(MAX_EX_HEAVY_RANGE)
-GLOBAL_VAR_INIT(MAX_EX_LIGHT_RANGE, 14)
+GLOBAL_VAR_INIT(MAX_EX_LIGHT_RANGE, 70)

```

Comment:
What the fuck dude, this is literally, LITERALLY nearly half the station.

Reply:
kek

Reply:
Awesome, right?